driver/csiplugin/controllerserver.go

```diff
@@ -899,37 +1039,28 @@ func (cs *ScaleControllerServer) CreateSnapshot(ctx context.Context, req *csi.Cr
 
 	snapName := req.GetName()
 
-	glog.V(5).Infof("CreateSnapshot - check if snapshot [%s] exist in fileset [%s] under filesystem [%s]", snapName, filesetName, filesystemName)
 	snapExist, err := conn.CheckIfSnapshotExist(filesystemName, filesetName, snapName)
 	if err != nil {
-		return nil, status.Error(codes.Internal, fmt.Sprintf("CreateSnapshot - Unable to get the snapshot details. Error [%v]", err))
+		glog.Errorf("CreateSnapshot [%s] - Unable to get the snapshot details. Error [%v]", snapName, err)
+		return nil, status.Error(codes.Internal, fmt.Sprintf("Unable to get the snapshot details for [%s]. Error [%v]", snapName, err))
 	}
 
 	if !snapExist {
-		glog.V(5).Infof("CreateSnapshot - creating snapshot [%s] in fileset [%s] under filesystem [%s]", snapName, filesetName, filesystemName)
 		snaperr := conn.CreateSnapshot(filesystemName, filesetName, snapName)
+		glog.Errorf("CreateSnapshot [%s] - Unable to create snapshot. Error [%v]", snapName, snaperr)
 		if snaperr != nil {
-			return nil, status.Error(codes.Internal, fmt.Sprintf("CreateSnapshot - Unable to create snapshot. Error [%v]", snaperr))
+			return nil, status.Error(codes.Internal, fmt.Sprintf("Unable to create snapshot [%s]. Error [%v]", snapName, snaperr))
```

Comment:
as per my vscode, error msg must start lower case letter and should not end with fullstop. I try to follow that. I use unable instead of Unable

Reply:
Done for instances related to this PR